src/server/pachyderm_test.go

```diff
@@ -9224,7 +9224,7 @@ func TestSpout(t *testing.T) {
 		t.Skip("Skipping integration tests in short mode")
 	}
 	c := getPachClient(t)
-	require.NoError(t, c.DeleteAll())
+	defer require.NoError(t, c.DeleteAll())
```

Comment:
What's the idea behind deferring this? Basically every other test cleans up the cluster at the start of the test rather than the end, because that guarantees that the test runs in a clean environment, and also that things from the test are still around afterward so that you can poke around the cluster to see what went wrong.

Reply:
Oh, running the spout test and having all the spouts continuously running was eating a ton of energy on my computer. I can change it back